records_mover/url/s3/s3_directory_url.py

```diff
@@ -3,6 +3,150 @@
 from ..base import BaseDirectoryUrl, BaseFileUrl
 from ..filesystem import FilesystemDirectoryUrl
 from typing import Dict, List
+import googleapiclient
+import json
+import time
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def _wait_for_transfer_job(project_id: str, job_name: str) -> None:
+    # TODO: Pass in correct creds
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker